_schema.graphql

```diff
@@ -5125,7 +5129,9 @@ type HomePageModuleContextSale implements Node {
   href: String
   name: String
   is_auction: Boolean
-  is_benefit: Boolean
+  is_benefit: Boolean @deprecated(reason: "Favor `isBenefit`")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 5132;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_benefit`